screens/ReportGenerationScreen.tsx

```diff
@@ -0,0 +1,177 @@
+import { isSmallScreen } from '../utils/helperFunc';
+
+const getStyles = (theme: NucaCustomTheme) =>
+  StyleSheet.create({
+    saveButton: {
+      height: 64,
+      maxWidth: 340,
+      minWidth: isSmallScreen() ? 64 : 340,
+    },
+    saveButtonContent: {
+      height: 64,
+      alignItems: 'center',
+      justifyContent: 'center',
+      flexDirection: 'row-reverse',
+    },
+    saveButtonLabel: {
+      fontSize: 18,
+      marginStart: 8,
+      textAlignVertical: 'bottom',
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.background,
+    },
+    loadingContainer: {
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    container: {
+      backgroundColor: theme.colors.background,
+      flex: 1,
+    },
+    scrollView: {
+      width: '100%',
+      height: '1000%',
+    },
+    contentContainer: {
+      padding: 20,
+      width: isSmallScreen() ? '100%' : '85%',
+    },
+    screenTitleContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+    },
+    screenTitleIcon: {
+      fontSize: 28,
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.text,
+    },
+    screenTitleLabel: {
+      fontSize: 18,
+      marginStart: 8,
+      textAlignVertical: 'bottom',
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.text,
+    },
+    pickerContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      flexWrap: 'wrap',
+    },
+    datePickerContainer: {
+      width: '50%',
+      flexDirection: 'column',
+      justifyContent: 'flex-start',
+      alignItems: 'flex-start',
+      padding: 4,
+    },
+    textInputTitle: {
+      color: theme.colors.text,
+      fontSize: 12,
+      marginTop: 20,
+      fontFamily: 'Nunito_700Bold',
+      textTransform: 'uppercase',
+    },
+    titleRow: {
+      flexDirection: 'row',
+      alignItems: 'baseline',
+      paddingBottom: 12,
+    },
+  });
+
+export const ReportGenerationScreen = (
+  _params: NativeStackScreenProps<RootStackParamList, 'ReportGeneration'>
+) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const [isInProgress, _setIsInProgress] = useState(false);
+
+  return (
+    <>
+      <Appbar forDetailScreen />
+      <View style={styles.container}>
+        <ScrollView style={styles.scrollView}>
+          <View
+            style={{
+              alignItems: 'center',
+            }}
+          >
+            <View style={styles.contentContainer}>
+              <View style={styles.screenTitleContainer}>
+                <View style={styles.titleRow}>
+                  <Headline style={styles.screenTitleIcon}>
+                    <View>
+                      <Icon source={'file-document-outline'} size={22} />
+                    </View>
+                  </Headline>
+                  <Caption style={styles.screenTitleLabel}>
+                    Generează raport
+                  </Caption>
```

Comment:
The icon and the caption are not aligned horizontally as seen in the screen recording. Can you adjust this?